scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -4,6 +4,6 @@ module.exports = {
   // Keep the MAX_SIZE +5 above the largest value and MIN_SIZE -5
   // below the smallest value in the build output; this avoids the
   // need for frequent changes as bundle sizes fluctuate.
-  MIN_SIZE: 568,
-  MAX_SIZE: 855,
+  MIN_SIZE: 574,
+  MAX_SIZE: 966,
```

Comment:
Shame this adds so much to the bundle size. Great idea only adding this weight to the page types that use Optimizely though 🙂